data/districts_data.js

```diff
@@ -105,7 +105,7 @@ var districts_data = {
2209287739981 ], [ 31.56260992289981, -0.775482142645138 ], [ 31.576020918889888, -0.786068772131615 ], [ 31.576318797689026, -0.801815372151529 ], [ 31.58171472447145, -0.832460141319466 ], [ 31.585765771122574, -0.864388857836337 ], [ 31.58558349909903, -0.884838514583346 ], [ 31.583875928119067, -0.896948498600874 ], [ 31.579461596154832, -0.906705822109374 ], [ 31.563930802947603, -0.929491715758772 ], [ 31.553719318038453, -0.938242534204717 ], [ 31.537241026430923, -0.948980335713714 ], [ 31.518020273830103, -0.957817428031812 ], [ 31.501251029487548, -0.976269322951091 ], [ 31.487812569971634, -0.987695082700559 ], [ 31.477076676696687, -1.000035349403071 ], [ 31.571469894057785, -1.001624572750864 ], [ 31.733431677945024, -0.999638198649322 ], [ 31.8009542440454, -0.999845673412505 ], [ 31.999898977376215, -0.999912568124114 ], [ 32.000506994219975, -0.81868538463129 ], [ 31.892175900125835, -0.796377443394242 ], [ 31.800992575444322, -0.752807606277819 ], [ 31.791913224009789, -0.752832221965809 ], [ 31.765123059732471, -0.741427731150057 ], [ 31.751304080249312, -0.734323391814355 ], [ 31.728539573419944, -0.710700227912937 ], [ 31.720863322978136, -0.69957266649098 ], [ 31.716434073480979, -0.677890488659984 ], [ 31.707216390109533, -0.659870356409505 ], [ 31.699972115422803, -0.656856475841031 ], [ 31.679782796531835, -0.625058299340076 ], [ 31.675978100930369, -0.610865872483449 ], [ 31.678364385450514, -0.594639899069319 ], [ 31.687053444225988, -0.584457372945645 ], [ 31.68270172943733, -0.581833313265094 ], [ 31.687721963219531, -0.560922571555762 ], [ 31.691193169149223, -0.557233000929405 ], [ 31.683252965531935, -0.548973210273178 ], [ 31.707103181263125, -0.531847152931626 ], [ 31.698910396323821, -0.526454616135597 ], [ 31.684367890877091, -0.524009424707428 ], [ 31.675468885446126, -0.515333478889836 ], [ 31.667074529910558, -0.514544828319309 ], [ 31.658232726054514, -0.499655162605963 ], [ 31.659256646197818, -0.491451689518171 ], [ 31.665068414364381, -0.485202856970893 ], [ 31.666202734450401, -0.460673265037868 ], [ 31.662469486338317, -0.457705982019319 ], [ 31.657494469548116, -0.444255368052131 ], [ 31.6450912821076, -0.439739748171799 ], [ 31.621398699941857, -0.4441944432309 ], [ 31.596211131486037, -0.431310177710301 ], [ 31.590528906212292, -0.443021869703721 ], [ 31.577766198958997, -0.453140260303431 ], [ 31.571514000522264, -0.463115119992555 ], [ 31.569550110633354, -0.476898540757126 ], [ 31.563916984443175, -0.482324059351246 ], [ 31.540162121762862, -0.495668507349817 ], [ 31.525529564789242, -0.497772595825829 ], [ 31.503734074013213, -0.516733431806072 ], [ 31.478175614869048, -0.543687892024146 ], [ 31.466744788209212, -0.547546812593445 ], [ 31.460677686548706, -0.554708148414823 ], [ 31.44526019555391, -0.554667936674596 ], [ 31.438517441597433, -0.562498213080521 ], [ 31.441223980421285, -0.57971865006342 ], [ 31.444336838857144, -0.586140738788156 ], [ 31.439953578850439, -0.5980865880018 ] ] ] } },
```

Comment:
Hi @lynnnakiganda ,

what changed precisely? On this line?

Reply:
I changed kyotara to kyotera

Reply:
I changed kyotara to kyotera